vagrant/verify-install.sh

```diff
@@ -0,0 +1,5 @@
+#tests to see if vagrant has already been installed via Construct
+test -d $HOME/Applications/"Vagrant.app"
```

Comment:
This can fail and your script with still exit zero.  You can simplify these by adding `&& exit 1` to the end of each test